mcgill-augmented-reality-frontend/screens/SignupScreen.js

```diff
@@ -0,0 +1,116 @@
+import { React, useState } from 'react';
+import { StyleSheet, View, TouchableOpacity, TouchableWithoutFeedback, Text, TextInput, Keyboard } from 'react-native';
+import { registerUser } from '../api';
+
+function SignupScreen({ navigation }) {
+
+    const [name, onChangeName] = useState('');
+    const [email, onChangeEmail] = useState('');
+    const [password, onChangePassword] = useState('');
+
+    function handlePress() {
+        registerUser(email, password)
+        .then(res => {
+            if (res.status !== 200) {
```

Comment:
When you create a user the status that is returned from the backend is a 201 so this will throw and error when it doesn't have to. I think it'd be better to check if(!res.ok) where ok is if the request was successful (between 200 and 299)